### PR TITLE
Use I18n translations for custom labels on dashboards

### PR DESCRIPTION
**Pull request by graysonwright**
## Problem:

There was uncertainty about how to create custom labels for fields. In https://github.com/thoughtbot/administrate/issues/451, it was suggested that we could use I18n translations to solve the problem.

If we accept that as the standard way to set custom labels, we should have a test for it.

## Solution:

- Add a test to verify that labels can be customized through I18n.
- Add documentation about how to customize labels

## To Do:

- [x] Update view code so the same translation definition applies on the show and collection pages, in addition to the form page.

### CODE REVIEW

File: app/views/administrate/application/_collection.html.erb [hunk 1]
```diff
@@ -22,13 +22,17 @@ to display a collection of resources in an HTML table.
   <thead>
     <tr>
       <% collection_presenter.attribute_types.each do |attr_name, attr_type| %>
-        <th class="cell-label cell-label--<%= attr_type.html_class %>
-          cell-label--<%= collection_presenter.ordered_html_class(attr_name) %>
+        <th class="cell-label
+        cell-label--<%= attr_type.html_class %>
+        cell-label--<%= collection_presenter.ordered_html_class(attr_name) %>
```
Comment (graysonwright) [inline app/views/administrate/application/_collection.html.erb]: Unrelated change: improve spacing of a super-long HTML tag

**Comment by graysonwright** [comment]
@mcmire, can I get a review? Thanks!

**Comment by kuboon** [comment]
Administrate already use ``Model.human`` to translate model name.
How about using ``Model.human_attribute_name`` for attributes?
http://apidock.com/rails/ActiveModel/Translation/human_attribute_name
